tests/kubectl_test.go

```diff
@@ -130,34 +130,12 @@ var _ = Describe("[sig-compute]oc/kubectl integration", func() {
 			Expect(resultFields[len(expectedHeader)]).To(Equal(vm.Name))
 			// Verify one of the wide column output field
 			Expect(resultFields[len(resultFields)-verifyPos]).To(Equal(expectedData))
+
 		},
-			table.Entry("[test_id:3468]virtualmachine", "get", "vm", "wide", []string{"NAME", "AGE", "VOLUME", "CREATED"}, 1, "true"),
+			table.Entry("[test_id:3468]virtualmachine", "get", "vm", "wide", []string{"NAME", "AGE", "STATUS", "VOLUME", "CREATED"}, 1, "true"),
 			table.Entry("[test_id:3466]virtualmachineinstance", "get", "vmi", "wide", []string{"NAME", "AGE", "PHASE", "IP", "NODENAME", "LIVE-MIGRATABLE", "PAUSED"}, 1, "True"),
 		)
 
-		table.DescribeTable("should verify set of wide columns for", func(verb, resource, option string, expectedHeader []string, verifyPos int, expectedData string) {
-
```

Comment:
Was that a duplicated test? Or was it removed for something related to this PR?

Reply:
Yes, it was byte-identical to the test case defined just before it.

Reply:
Thanks. Strange that Sonar cloud didn't pick it :(

Reply:
Indeed SonarCloud is configured to exclude `*_test.go` files. As if they don't need to be maintained :-)